x-pack/test/functional/services/machine_learning/api.ts

```diff
@@ -79,5 +141,69 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
     async cleanMlIndices() {
       await this.deleteIndices('.ml-*');
     },
+
+    async getJobState(jobId: string): Promise<JobState | undefined> {
+      log.debug(`Fetching job state for job ${jobId}`);
+      const jobStats = await esSupertest
+        .get(`/_ml/anomaly_detectors/${jobId}/_stats`)
+        .expect(200)
+        .then((res: any) => res.body);
+
+      expect(jobStats.jobs).to.have.length(1);
+      const state = jobStats.jobs[0].state as JobState;
```

Comment:
type assertion isn't needed here. this could be:
`const state: JobState = jobStats.jobs[0].state;` 
or `const state: JOB_STATE = jobStats.jobs[0].state;` if the previous suggested change is made.

Reply:
Done in f32ca31